src/js/events/SystemLogActions.js

```diff
@@ -181,16 +203,11 @@ const SystemLogActions = {
           subscriptionID
         });
       }
-      // Clean up event listeners
-      source.removeEventListener('message', messageListener);
-      source.removeEventListener('error', errorListener);
-      // Close connection,
-      // no need to delete source as the reference is not stored
-      source.close();
+
+      unsubscribe(subscriptionID);
     }
 
-    source.addEventListener('message', messageListener, false);
-    source.addEventListener('error', errorListener, false);
+    subscribe(url, subscriptionID, messageListener, errorListener);
```

Comment:
When using range, we don't need to store the source! I guess this will work, but again I would add the `Date.now()` to ensure uniqueness